html/index.html

```diff
@@ -233,20 +233,24 @@ <h1>Sampling and Sequencing Simulator</h1>
   <input type=radio name=sampling id=custom_sampling><label for=custom_sampling>Custom</label>
   <tr><td>Shedding amount:
     <td><textarea rows=5 cols=40 id=shedding_values></textarea>
-    <td align=right>CV(log):<td><input type=text id=cv_log_shedding_values>
+    <td align=right>&#963;<sub>g</sub>:<td><input type=text id=geom_stdev_shedding_values>
 
-<tr><td colspan=4 class=note>If you ran metagenomic sequencing on a sample
-that was from just a single infected person, what fraction of
+<tr><td colspan=4 class=note>If you ran metagenomic sequencing on a
+sample that was from just a single infected person, what fraction of
 sequencing reads would be from the pathogen?  This could be a single
 number to model all infected people as shedding an equal amount, or
 multiple space-separated numbers to represent a set of equally likely
-possible shedding amounts.
+possible shedding amounts. For &#963;<sub>g</sub> you specify the
+geometric standard deviation of per-simulation noise representing our
+uncertainty about whether the true relative abundance would be higher
+or lower.
 <tr><td>Shedding duration (days):
     <td><input type=text id=shedding_duration value=5>
-    <td align=right>CV:<td><input type=text id=cv_shedding_duration>
+<td align=right>CV<sub>g</sub>:<td><input type=text id=cv_shedding_duration>
 <tr><td colspan=4 class=note>Once someone is infected, how many days do they
 shed for?  This is a simplified model where we assume people shed
-equally throughout their infection.
+equally throughout their infection. For CV<sub>g</sub> specify the
+geometric coefficient of variation.
```

Comment:
Should be standard deviation? (Geometric mean = 0)

Reply:
The code this describes is:

```
function get_input_geom_cv(input, input_geom_cv) {
  const geom_mean = get_input(input);
  const geom_cv = get_input(input_geom_cv) / 100;

  if (geom_cv < epsilon) {
    return geom_mean;
  }

  const geom_stdev = geom_cv * geom_mean;
  return lognormal(geom_mean, geom_stdev);
}
```

Unlike with $σ_g$ on the shedding amount, the mean that $CV_g$ applies to is the number of shedding days isn't zero.

Reply:
Thanks for flagging this!  The function is lognormal(mu, sigma) and https://en.wikipedia.org/wiki/Log-normal_distribution says mu and sigma "are the expected value (or mean) and standard deviation of the variable's natural logarithm, not the expectation and standard deviation of [the variable] itself"

I think your correction is right if `mu = arithmatic_mean(log(X)) = log(geometric_mean(X))`.

Then for sigma I think it's easiest if I just have people specify it directly.  Done in ca63c42!

Reply:
Also, stop using percentages, now that it's sigma and not CV: a52fa1d